### ISSUE TITLE
Rewrite report system

### ISSUE THREAD

**Issue by IntinteDAO** [koraykarakus/steemnova-1.8-x#1]
I think one project you should sit down on is a rewrite of the fight reporting system. First of all, it's terribly suboptimal because it stores the entire fight record as text instead of digital values, making it take up a lot of disk space. Only from these values should PHP create text.

**Comment by koraykarakus** [comment]
true, it is a good idea, working on admin panel's autoload on my free time, maybe after autoload update, i will work on reports. By the way, thank you for your steemnova project, it was a good starting point, i tried to send some pull req. for it but you were inactive for so long, so i started working on this :)

**Comment by IntinteDAO** [comment]
Thanks. I would elaborate further, but people have exploits and the game has bugs. I'll try to rewrite the bot one day (if I get the chance) from PHP to Python + Selenium, I think it will be much better.

**Comment by fossil2** [comment]
Ich kann  euch nur danken für eure tolle Arbeit.
